pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/NonPersistentTopics.java

```diff
@@ -129,6 +129,18 @@ public void createPartitionedTopic(@PathParam("property") String property, @Path
             throw new RestException(e);
         }
     }
+
+    @PUT
+    @Path("/{property}/{cluster}/{namespace}/{destination}/unload")
+    @ApiOperation(value = "Unload a topic")
+    @ApiResponses(value = { @ApiResponse(code = 403, message = "Don't have admin permission") })
```

Comment:
@ApiResponse(code = 404, message = "Topic does not exist"),

Reply:
yes, fixed it.